### DISCUSSION TITLE
TIMx->SMCR.SMS[3] missing in peripheral view for STM32L4R5

### DISCUSSION THREAD

**Discussion by PerennialNovice** [stm32duino/Arduino_Core_STM32#2397] [General]
I know you don't support sloeber, but Iam quiet confident this is an issue not related to sloeber alone.

When debugging timers, I noticed that TIMx->SMCR is missing bit 16 for SMS[3] in the SMCR bitfield list:
![image](https://github.com/stm32duino/Arduino_Core_STM32/assets/85557472/d7cd615a-e703-49bf-b04c-6f1117dc9559)

the SMCR value gets updated correctly, but the corresponding bit for SMS[3] is missing.
Would it be possible to change SMS[2:0] to SMS[3:0] ?

By editing the .svd-file I managed to get a field SMS3 at bit 16, but it would be nice to have ist as one field. But I have no idea if the .svd format allows for this...

**Comment by fpistm** [comment]
Please report this to STMicroelectronics GitHub. Not related to Arduino stuff.
